Shared/Samples/Augment reality to collect data/AugmentRealityToCollectDataView.swift

```diff
@@ -0,0 +1,204 @@
+            url: URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/AR_Tree_Survey/FeatureServer/0")!
+        )
+        /// The graphics overlay which shows marker symbols.
+        @State var graphicsOverlay: GraphicsOverlay = {
+            let graphicsOverlay = GraphicsOverlay()
+            let tappedPointSymbol = SimpleMarkerSceneSymbol(
+                style: .diamond,
+                color: .orange,
+                height: 0.5,
+                width: 0.5,
+                depth: 0.5,
+                anchorPosition: .center
+            )
+            graphicsOverlay.renderer = SimpleRenderer(symbol: tappedPointSymbol)
+            graphicsOverlay.sceneProperties.surfacePlacement = .absolute
+            return graphicsOverlay
+        }()
+        /// The selected tree health for the new feature.
+        @State private var treeHealth: TreeHealth?
+        
+        init() {
+            let featureLayer = FeatureLayer(featureTable: featureTable)
+            featureLayer.sceneProperties.surfacePlacement = .absolute
+            scene.addOperationalLayer(featureLayer)
+        }
+        
+        /// Adds a feature to represent a tree to the tree survey service feature table.
+        /// - Parameter treeHealth: The health of the tree.
+        func addTree(health: TreeHealth) async throws {
+            guard let featureGraphic = graphicsOverlay.graphics.first,
+                  let featurePoint = featureGraphic.geometry as? Point else { return }
+            
+            // Create attributes for the new feature.
+            let featureAttributes: [String: Any] = [
+                "Health": health.rawValue,
+                "Height": 3.2,
+                "Diameter": 1.2
+            ]
+            
+            if let newFeature = featureTable.makeFeature(
+                attributes: featureAttributes,
+                geometry: featurePoint
+            ) as? ArcGISFeature {
+                do {
+                    // Add the feature to the feature table.
+                    try await featureTable.add(newFeature)
+                    _ = try await featureTable.applyEdits()
+                } catch {
+                    throw error
+                }
+                newFeature.refresh()
+            }
+            
+            graphicsOverlay.removeAllGraphics()
+        }
+    }
+}
+
+private extension AugmentRealityToCollectDataView {
+    /// The health of a tree.
+    enum TreeHealth: Int16, CaseIterable, Equatable {
+        /// The tree is dead.
+        case dead = 0
+        /// The tree is distressed.
+        case distressed = 5
+        /// The tree is healthy.
+        case healthy = 10
+        
+        /// A human-readable label for each kind of tree health.
+        var label: String {
+            switch self {
+            case .dead: return "Dead"
+            case .distressed: return "Distressed"
+            case .healthy: return "Healthy"
```

Comment:
Not a big deal, but `return` keywords aren't necessary. You could omit them if you want.